### PR TITLE
Update of PoseEstimation model

### PR DESCRIPTION
**Pull request by BloodAxe**
* New model weights
* Recipe to train DEKR-W32-NO-DC 
* Documentation
* Rescoring 
* FlipTTA

### CODE REVIEW

File: documentation/source/PoseEstimation.md [hunk 1]
```diff
@@ -407,3 +407,47 @@ See [DEKRPoseEstimationDecodeCallback](https://docs.deci.ai/super-gradients/docs
...
+
+To train a rescoring model you need to generate the training data first. This assumes that you have a pretrained pose estimation model.
+To generate the dataset for rescoding model we run inference on the original dataset (COCO in this example) using our pretrained pose estimation model and save it's predictions to Pickle files. 
+
+The rescoring model input are poses `[B,J,3]` and the outputs are the rescoring scores `[B,1]`. The targets are computed object-keypoint similarity (OKs) scores between predicted pose and ground-truth pose.
+
+Currently, rescoring is only supported for DEKR architecture. 
+
+```bash
+python -m super_gradients.script.generate_rescoring_training_data --config-name=script_generate_rescoring_data_dekr_coco2017 rescoring_data_dir=OUTPUT_DATA_DIR  checkpoint=PATH_TO_TRAINED_MODEL_CHECKPOINT`.
```
Comment (shaydeci) [inline documentation/source/PoseEstimation.md:438]: scripts* ?

File: documentation/source/PoseEstimation.md [hunk 2]
```diff
@@ -407,3 +407,47 @@ See [DEKRPoseEstimationDecodeCallback](https://docs.deci.ai/super-gradients/docs
...
+
+```bash
+python -m super_gradients.train_from_recipe --config-name coco2017_pose_dekr_rescoring \
+  dataset_params.train_dataset_params.pkl_file=OUTPUT_DATA_DIR/rescoring_data_train.pkl \
+  dataset_params.val_dataset_params.pkl_file=OUTPUT_DATA_DIR/rescoring_data_valid.pkl
+```
+
+This recipe uses custom callback to compute pose estimation metrics on the validation dataset using coordinates of poses from step 1 and confidence values after rescoring.
+
+See `tests.integration_tests.pose_estimation_dataset_test.PoseEstimationDatasetIntegrationTest.test_dekr_model_with_rescoring` test for more details and end-to-end usage example.
```
Comment (shaydeci) [inline documentation/source/PoseEstimation.md:453]: Maybe add some link.

**Review by shaydeci — CHANGES_REQUESTED** [review]
Looks great!
Just found 2 typos in the docs, see inline.
Then LGTM on my end.

File: documentation/source/PoseEstimation.md (hunk 2 above)
Comment (BloodAxe) [reply documentation/source/PoseEstimation.md:453]: Nice catch, thanks.
Fixed in 4dd1bbf4d86fdf6d0a693f2388e29409a838340e

File: documentation/source/PoseEstimation.md (hunk 1 above)
Comment (BloodAxe) [reply documentation/source/PoseEstimation.md:438]: 4dd1bbf4d86fdf6d0a693f2388e29409a838340e

File: documentation/source/PoseEstimation.md (hunk 2 above)
Comment (shaydeci) [reply documentation/source/PoseEstimation.md:453]: actually meant to add a link instead of just the dotparh, would be a bit more friendly to the reader (:

Comment (BloodAxe) [reply documentation/source/PoseEstimation.md]: Ah, pardom me. Updated with a link ;)
